wikimedia/wikipedia-ios
# T378019 yir landscape

**Phabricator:** 
https://phabricator.wikimedia.org/T378019

### Notes
* Locks the YIR into 

### Test Steps
1. Open app normally, but in one of the targeted regions/languages in order to show Year in Review. 
2. Ensure you have at least 6+ articles in history, background and foreground the app.
3. Navigate to any article, test to ensure landscape mode works. 
4. Return to portrait mode.
5. Navigate to Year in Review, try landscape mode.
6. Ensure it doesn't work.
7. Navigate OUT of Year in Review, ensure you are able to get portrait and landscape. 

### Screenshots/Videos
